playgrounds/autocomplete/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@meilisearch/autocomplete-playground",
+  "version": "0.0.0",
```

Comment:
Why not v0.1.0 ?

Reply:
It's a playground and not a library. As per the turbo's convention, they should keep the `0.0.0` version.